scripts/README.md

```diff
@@ -1,21 +1,11 @@
 scripts
 =======
 
```

Comment:
More detail should be added to this article to describe the specific scripts.

Reply:
added more information. These are not supposed to be run by people after all, so i have adjusted the texts. Check it out

Reply:
Looks good to me. Please review.